examples/app-with-service-account/go.mod

```diff
@@ -0,0 +1,95 @@
+module github.com/grafana/app-with-service-account
+
+go 1.19
+
+require (
+	github.com/go-jose/go-jose/v3 v3.0.0
+	github.com/grafana/grafana-plugin-sdk-go v0.185.1-0.20231018140535-4e998bd199d8
```

Comment:
Currently pointing to a commit from https://github.com/grafana/grafana-plugin-sdk-go/pull/777 to get the BearerAuthMiddleware. This needs to be fixed before merging.